grammars/css.cson

```diff
@@ -336,176 +1214,367 @@
         'include': '#media-query'
       }
     ]
-  'numeric-values':
+  'media-types':
+    'captures':
+      '1':
+        'name': 'support.constant.media.css'
+      '2':
+        'name': 'invalid.deprecated.constant.media.css'
+    'match': '''(?xi)
+      (?<=^|\\s|,|\\*/)
+      (?:
+        # Valid media types
+        (all|print|screen|speech)
+        |
+        # Deprecated in Media Queries 4: http://dev.w3.org/csswg/mediaqueries/#media-types
+        (aural|braille|embossed|handheld|projection|tty|tv)
+      )
+      (?=$|[{,\\s;]|/\\*)
+    '''
+  'nested-selector':
```

Comment:
I don't foresee any changes to the syntax. Every developer is already familiar with the `&`-notation used in Sass and Less, and in the off-chance a revision is made, the rule can be easily amended.